movie_app_react/src/modules/moviebrowser/moviecard/MovieCard.jsx

```diff
@@ -35,9 +77,10 @@ class MovieCard extends React.Component {
                             <div>
                                 <h4 className="movie-title">{movie.movieTitle}</h4>
                                 <p className="duration-genre">{movie.movieDuration} - <EllipsisText text={movie.movieGenre} length={25}/> </p>
-                                <a className="movie-trailer" target='_blank' href={ movie.movieTrailerUrl } rel="noopener noreferrer">
+                                <a className="movie-trailer" href={ movie.movieTrailerUrl } >
```

Comment:
Can you add these properties back? This is to open the Trailer link in a new tab

Reply:
Done